README.md

```diff
@@ -1 +1,29 @@
-# hw01
\ No newline at end of file
+# hw01
+
+# Brief biography 
+## Introduction
+My name is Ahyeon Cho (*Ah* is pronounced like the *o* in the word otter, *yeon* is pronounced y + uh + n). I am from South Korea.
+
+## Interesting things to know about me
+
+1. I am a die-hard **Friends** fan
+2. I do not like *steamed* carrots 
+3. But I do like *fresh* carrots
+4. I memorize almost all **90s disney songs** by heart
+5. The follwing image is what my life has been like since quarantine
+
+![Addiction to Zelda](https://pmcvariety.files.wordpress.com/2018/04/lzbotw_tvc_2.jpg)
+
+## Reasons for taking this course
+I am *very* technologically challenged. I learned much later compared to my peers how to capture an image on my phone. The following are the few things that I am able to do using my computer:
+
+- search on Google
+- read and send e-mail
+- use word and powerpoint 
+- watch Youtube videos
```

Comment:
I notice that in this list the words are not capitalized at the beginning of each line, while in this previous list they are. This inconsistency just a tiny detail that may not be as important here, but will be _very_ important when we start paying attention to coding style in future homework assignments